crates/red_knot_python_semantic/resources/mdtest/type_api.md

```diff
@@ -350,3 +371,37 @@ def type_of_annotation() -> None:
 # error: "Special form `knot_extensions.TypeOf` expected exactly one type parameter"
 t: TypeOf[int, str, bytes]
 ```
+
+## `LiteralExt`
+
+`LiteralExt` creates (unions of) class, function or slice types.
```

Comment:
You can't use it for *unions* of literals, but I think you should be able to use `knot_extensions.TypeOf` for function literals and slice literals, in addition to class literals, as is demonstrated in the section above this one.

Reply:
I think I'd also prefer to stick with TypeOf for now rather than adding `LiteralExt`